README.md

```diff
@@ -1,3 +1,62 @@
-# juno-sdk
```

Comment:
Could we have a README header more like the one in Juno? Should be a relatively easy copy paste here with a few changes (you can leave off the badge showing the current status of tests)